admin_manual/configuration_server/activity_configuration.rst

```diff
@@ -0,0 +1,27 @@
+Configuring the Activity App
+============================
+
+You can configure your ownCloud server to automatically send out e-mail notifications
+to your users for various events like:
+
+* A file or folder has been shared
+* A new file or folder has been created
+* A file or folder has been changed
+* A file or folder has been deleted
+
+Users see actions (delete, add, modify) that happen to files they have access to.
+Sharing actions are only visible the the sharer and sharee.
+
+Enabling the Activity App
```

Comment:
The app is shipped, so I think it's enabled by default. Could be mentioned here I guess.